dataprofiler/profilers/profile_builder.py

```diff
@@ -421,27 +418,16 @@ def __init__(self, data, samples_per_update=None, min_true_samples=0,
         self._min_true_samples = min_true_samples
         self._profile = dict()
 
+        # matches structured data profile
+        # TODO: allow set via options
+        self._sampling_ratio = 0.2
+        self._min_sample_size = 5000
+
         if isinstance(data, data_readers.text_data.TextData):
             raise TypeError("Cannot provide TextData object to Profiler")
 
-        # assign data labeler
-        data_labeler_options = self.options.structured_options.data_labeler
-        if data_labeler_options.is_enabled \
-                and data_labeler_options.data_labeler_object is None:
-            
-            try:
-                
-                data_labeler = DataLabeler(
-                    labeler_type='structured',
-                    dirpath=data_labeler_options.data_labeler_dirpath,
-                    load_options=None)
-                self.options.set({'data_labeler.data_labeler_object': data_labeler})
-                
-            except Exception as e:
-                utils.warn_on_profile('data_labeler', e)
-                self.options.set({'data_labeler.is_enabled': False})
-
-        self.update_profile(data)
```

Comment:
moved to inside `update_profile`, was done initially in case merging of profiles could duplicate the labeler, but looks like merge handles it by passing its options through.

Reply:
Technically this allows a profile to be instantiated without creating the labeler until profile time making it faster. Not sure if this is desirable.